crates/client/block_production/src/lib.rs

```diff
@@ -201,8 +201,9 @@ impl<Mempool: MempoolProvider> BlockProductionTask<Mempool> {
     fn continue_block(
         &mut self,
         bouncer_cap: BouncerWeights,
-    ) -> Result<(StateDiff, VisitedSegments, BouncerWeights, ContinueBlockStats), Error> {
+    ) -> Result<(StateDiff, VisitedSegments, BouncerWeights, ContinueBlockStats, bool), Error> {
```

Comment:
At this point consider putting the return type behind a struct.

Reply:
sure thing, what's the standard for this though? more than 4 should be struct? just curious

Reply:
Not really any standard tbf. I guess maybe clippy might start complaining at a certain point but mostly just for readability purposes, as its easy to get confused regarding the meaning of each field in a return type if it is just a tupple.

Reply:
fair enough, will do